Feat: Implemented the home exploreCourses page

Iwueseiter opened the pull request:
Implemented the explore Course section according to the Figma without changing any other files.
closes #69 

![Screenshot 2025-07-25 225613](https://github.com/user-attachments/assets/b3bc1be0-cfed-4007-8cf7-c368a1051bde)

Jeffrey0522 commented:
@Iwueseiter Please see the figma. It isn't the correct component. In the issue description is the component pic

Jeffrey0522 commented:
@Iwueseiter please check issue description

Iwueseiter commented:
> @Iwueseiter please check issue description

Hello @Jeffrey0522, I think it's a bit of a mix-up. I've implemented the page but I didn't remove the screenshot of the previous one that I added in the PR description.

Jeffrey0522 commented:
@Iwueseiter All good. Thanks for your work!
